cdp_scrapers/instances/portland.py

```diff
@@ -134,8 +139,65 @@ def get_matter(self, event_page: BeautifulSoup) -> Optional[Matter]:
         #   The first description phrase in each item on agenda page.
         #   This is usually hyperlinked to another page with more details.
 
+        # Find document number
+        doc_number_element_sibling = event_page.find(
+            "div", text=re.compile("Document number"), attrs={"class": "field__label"}
+        )
+
+        # If there is no document number, skip this minute item
+        if doc_number_element_sibling == None:
+            return None
+        doc_number_element = doc_number_element_sibling.next_sibling
+        doc_number = doc_number_element.find("div", class_="field__item").text.strip()
+
+        # Find title
+        title_div = event_page.find("div", class_="council-document__title")
+        matter_title = title_div.find("a").text.strip()
+
+        # Find type
+        title_div.find("a").clear()
```

Comment:
What is the purpose/effect of calling `clear()` on the element returned by `find()`?

Reply:
The HTML on the Portland city government's website encodes the title of each matter item in the following format:
![image](https://user-images.githubusercontent.com/58454820/150667134-6d78d830-610b-47b3-bd8d-1fab0ee9670e.png)

Reply:
TIL [`Tag.clear()`](https://www.crummy.com/software/BeautifulSoup/bs4/doc/#clear). Thank you Shak! :sweat_smile: